data/strings/types_strings.txt

```diff
@@ -21621,6 +22021,46 @@
     zh-Hans = 摩托车商店
     zh-Hant = 機車行
 
+  [type.shop.motorcycle_repair]
+    en = Motorcycle Repair
+    ar = تصليح الدراجات النارية
+    be = Рамонт матацыклаў
+    bg = Ремонт на мотоциклети
+    ca = Reparació de motos
+    cs = Opravy motocyklů
+    da = Motorcykel reparation
+    de = Motorrad Reparatur
```

Comment:
"Motorradwerkstatt" would fit better in my opinion. "Wekstatt" means like repair shop.